README.md

```diff
@@ -1,3 +1,8 @@
+# This plugin is superseded by the OpenNMS Plugin for Grafana.
+
+Please use the [OpenNMS Plugin for Grafana](https://grafana.com/grafana/plugins/opennms-opennms-app/) instead.
+That plugin is compatible with current Grafana releases, and does not have a naming conflict with the package manager for Kubernetes.
```

Comment:
Should we call out the version of Grafana we are compatible with?

Reply:
👋 just a suggestion to remove ambiguity on which plugin is being referred to and to point users directly to the upgrade instructions

Reply:
Also, [Levitate](https://github.com/grafana/levitate) and the [is-compatible](https://github.com/grafana/plugin-actions/tree/main/is-compatible) GitHub action can be used to perform a basic compatibility check for Grafana versions if that is of interest to you.